test/basic.js

```diff
@@ -0,0 +1,106 @@
+var test = require('tape');
+var parse = require('../parse.js');
+var index = require('../index.js')
+
+/* BEGIN parse.js tests */
+
+test('parseResource playlist test 1', function (t) {
+	var 
+	  tagLine = "#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=240000",
+	  resourceLine = "prog_index.m3u8",
+	  manifestUri = "http://example.com/hls_manifest.m3u8",
+	  resource = parse.parseResource(tagLine, resourceLine, manifestUri);
+
+	t.plan(2);
+    t.equal(resource.type, 'playlist');
+    t.equal(resource.line, manifestUri + '/' + resourceLine);
+});
+
+test('parseResource playlist test 2', function (t) {
+	var 
+	  tagLine = "#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=440000",
+	  resourceLine = "prog_index2.m3u8",
+	  manifestUri = "http://example.com/hls_manifest.m3u8",
+	  resource = parse.parseResource(tagLine, resourceLine, manifestUri);
+
+    t.plan(2);
+    t.equal(resource.type, 'playlist');
+    t.equal(resource.line, manifestUri + '/' + resourceLine);
+});
+
+test('parseResource segment test 1', function (t) {
+	var 
+	  tagLine = "#EXTINF:10, no desc",
+	  resourceLine = "fileSequence0.ts",
+	  manifestUri = "http://example.com/hls_manifest.m3u8",
+	  resource = parse.parseResource(tagLine, resourceLine, manifestUri);
+    
+    t.plan(2);
+    t.equal(resource.type, 'segment');
+    t.equal(resource.line, manifestUri + '/' + resourceLine);
+});
+
+test('parseResource segment test 2', function (t) {
+	var 
+	  tagLine = "#EXTINF:10, no desc",
+	  resourceLine = "http://example.com/hls_manifest.m3u8/fileSequence0.ts",
+	  manifestUri = "http://example.com/hls_manifest.m3u8",
+	  resource = parse.parseResource(tagLine, resourceLine, manifestUri);
+
+    t.plan(2);
+    t.equal(resource.type, 'segment');
+    t.equal(resource.line, "http://example.com/hls_manifest.m3u8/fileSequence0.ts");
+});
+
+test('parseManifest test 1', function (t) {
```

Comment:
Test names should be statements that the test asserts. 

`parseManifest parses a manifest into the expected array of lines`